src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        readFromFile(fromFileName, stringBuilder);
+        String report = createReport(stringBuilder).toString();
+        writeToFile(toFileName, report);
+    }
+
+    private StringBuilder readFromFile(String fromFileName, StringBuilder stringBuilder) {
```

Comment:
Never return StringBuilder from methods - it's a service class
```suggestion
    private String readFromFile(String fromFileName) {
```